src/ol/source/tileutfgridsource.js

```diff
@@ -271,7 +271,7 @@ ol.source.TileUTFGridTile_.prototype.getImage = function(opt_context) {
  */
 ol.source.TileUTFGridTile_.prototype.getData = function(coordinate) {
   if (goog.isNull(this.grid_) || goog.isNull(this.keys_) ||
-      goog.isNull(this.data_)) {
+      !goog.isDefAndNotNull(this.data_)) {
```

Comment:
Yes, that would work as well, but the `handleLoad_` method only stores the loaded UTFGridJSON information at the moment and all the interpretation is done inside the `getData` method. This gives us possibility for different behavior for `null` and `undefined` values if this is needed in the future.